videopython/compose.py

```diff
@@ -0,0 +1,31 @@
+from videopython.transitions import Transition
+from videopython.video import Video
+from videopython.pipeline import TransformationPipeline
+
+
+class VideoComposer:
+    """
+    Composes multiple Videos into single video using selected transformations
+    on each video and applies transitions.
+    """
+
+    def __init__(self, transformation_pipeline: TransformationPipeline,
+                 transition: Transition):
+        self.transition = transition
```

Comment:
It can be worth adding transition type as a parameter. It can be either same transition for entire video or list of transition with length equal to number of videos.

Reply:
Good idea! Done :)